spec/spec_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
+require 'simplecov'
+SimpleCov.start
```

Comment:
Fixed in the latest commit, which uses the 'rails' profile that excludes `/config/` and `/test/` by default.